docs/tutorial.md

```diff
@@ -283,28 +325,44 @@ $ jj new puqltutt  # Replace the ID by what you have for B2
 Working copy now at: zxoosnnp c7068d1c (conflict) (empty) (no description set)
 Parent commit      : puqltutt f7fb5943 (conflict) B2
 Added 0 files, modified 0 files, removed 1 files
+There are unresolved conflicts at these paths:
+file1    2-sided conflict
+
 $ jj st
 The working copy is clean
 There are unresolved conflicts at these paths:
 file1    2-sided conflict
 Working copy : zxoosnnp c7068d1c (conflict) (empty) (no description set)
 Parent commit: puqltutt f7fb5943 (conflict) B2
+To resolve the conflicts, start by updating to it:
+  jj new puqltutt
+Then use `jj resolve`, or edit the conflict markers in the file directly.
+Once the conflicts are resolved, you may want to inspect the result with `jj diff`.
+Then run `jj squash` to move the resolution into the conflicted commit.
+
 $ cat file1
-<<<<<<<
-%%%%%%%
--b1
-+a
-+++++++
-b2
->>>>>>>
+<<<<<<< Conflict 1 of 1
+%%%%%%% Changes from base to side #1
+-b1+a+++++++ Contents of side #2
+b2>>>>>>> Conflict 1 of 1 ends
```

Comment:
I think this is a formatting mistake, it should be:

```
<<<<<<< Conflict 1 of 1
%%%%%%% Changes from base to side #1
-b1
+a
+++++++ Contents of side #2
b2
>>>>>>> Conflict 1 of 1 ends
```

Reply:
It could be good to file a bug with nushell, since it's pretty standard for echo to add a newline as far as I'm aware.